lib/namma-dsl/src/NammaDSL/DSL/Parser/API.hs

```diff
@@ -198,20 +206,23 @@ extractComplexTypeImports' = do
             )
             tps
 
-apiParser' :: ApiRead -> FilePath -> IO Apis
+apiParser' :: ApiRead -> FilePath -> IO (Apis, Apis)
 apiParser' apiRead filepath = do
   contents <- BS.readFile filepath
   case Yaml.decodeEither' contents of
     Left _ -> error "Not a Valid Yaml"
     Right yml -> do
       let extraParseInfo = ExtraParseInfo yml []
           apiState = ApiState def extraParseInfo
-      _apisRes <$> evalParser parseApis' apiRead apiState
+      -- Hack for remove qualified name of current module inside of api types
+      apiDef <- _apisRes <$> evalParser (parseApis' OTHER_PARSER) apiRead apiState
+      apiDefApiTypes <- _apisRes <$> evalParser (parseApis' API_TYPES_PARSER) apiRead apiState
```

Comment:
Don't like this part. But in current structure I need to run separate parser, if I want to change qualified names for current generator type. Need some refactor to fix this. Making types qualified should be separated from parsing